app/kubernetes/views/applications/applications.html

```diff
@@ -22,11 +22,6 @@
               >
               </kubernetes-applications-datatable>
             </uib-tab>
-            <uib-tab index="1" classes="btn-sm" select="ctrl.selectTab(1)">
-              <uib-tab-heading class="vertical-center"> <pr-icon icon="'svg-dataflow'"></pr-icon> Port mappings </uib-tab-heading>
-              <kubernetes-applications-ports-datatable dataset="ctrl.state.ports" table-key="kubernetes.applications.ports" order-by="Name" refresh-callback="ctrl.getApplications">
```

Comment:
state.ports might no longer be needed in applicationsController.js

Reply:
in addition, the portMappingsFromApplications function seems to only be used by the applicationsController.js, so can also be removed

Reply:
it is used elsewhere unfortunately